web/src/features/charts/DataSources.tsx

```diff
@@ -90,13 +85,11 @@ export function DataSources({
 
 function EmissionFactorTooltip({ t }: { t: TFunction<'translation', undefined> }) {
   return (
-    <Portal.Root className="pointer-events-none absolute left-0 top-0 z-50 flex h-full w-full flex-col content-center items-center justify-center bg-black/20 pb-40">
+    <Portal.Root className="pointer-events-none absolute left-0 top-0 z-50 flex h-full w-full flex-col content-center items-center justify-center gap-2 bg-black/20 pb-40">
       <div className="dark:border-1 relative mx-6 h-auto min-w-64 rounded-xl border bg-zinc-50 p-4 text-left text-sm opacity-80 shadow-md dark:border-gray-700 dark:bg-gray-900">
         {t('country-panel.emissionFactorDataSourcesTooltip')}
       </div>
-      <button className="p-auto pointer-events-auto mt-2 flex h-10 w-10 items-center justify-center self-center rounded-full border bg-zinc-50 text-black shadow-md">
-        <X />
-      </button>
+      <Button icon={<X />} type="secondary" backgroundClasses="pointer-events-auto" />
```

Comment:
@Alportan I know we have discussed this button before (the white button) but I decided to switch it to use the button component here and therefore it will follow the theme.

I think it should be fine as it has an outline and it will help with keeping things uniform as other tooltips don't have this white icon.

I will also switch other tooltips to use the button component so it's uniform in both style and functionality.